cspdk/sin300/tech.py

```diff
@@ -1,52 +1,47 @@
 """Technology definitions."""
 
 import sys
+from collections.abc import Iterable
 from functools import partial
 from typing import cast
 
 import gdsfactory as gf
-from gdsfactory.cross_section import get_cross_sections
+from gdsfactory.cross_section import CrossSectionSpec, LayerSpec, get_cross_sections
+from gdsfactory.routing.route_bundle import OpticalManhattanRoute
 from gdsfactory.technology import (
     LayerLevel,
     LayerMap,
     LayerStack,
     LayerViews,
     LogicalLayer,
 )
-from gdsfactory.typings import ConnectivitySpec, Layer
+from gdsfactory.typings import ComponentSpec, ConnectivitySpec, Layer
 
 from cspdk.sin300.config import PATH
 
 nm = 1e-3
 
 
 class LayerMapCornerstone(LayerMap):
-    WG: Layer = (3, 0)
-    SLAB: Layer = (5, 0)
-    FLOORPLAN: Layer = (99, 0)
-    HEATER: Layer = (39, 0)
-    GRA: Layer = (6, 0)
-    LBL: Layer = (100, 0)
-    PAD: Layer = (41, 0)
-    NITRIDE: Layer = (203, 0)
-    NITRIDE_ETCH: Layer = (204, 0)
+    # TODO: how can we make this pass type checking?
```

Comment:
remove?

Reply:
no, that makes pyright error out (not a huge deal, but I like my repositories type clean)